2.13.15.md

```diff
@@ -0,0 +1,76 @@
+
+### Compatibility
+
+* Support JDK 23 in optimizer (via upgrade to patched ASM 9.7.0) ([#10744](https://github.com/scala/scala/pull/10744)
+
+### Features
+
+* Support alternative JVM system images (via `-system`, like javac) ([#10801](https://github.com/scala/scala/pull/10801) by [@szeiger](https://github.com/szeiger))
+* Add `-Ycollect-statistics` for collecting statistics without printing them ([#10795](https://github.com/scala/scala/pull/10795) by [@szeiger](https://github.com/szeiger))
+
+### Align with Scala 3
+
+* Support Scala 3.5 in TASTy reader ([#10811](https://github.com/scala/scala/pull/10811) by [@bishabosha](https://github.com/bishabosha))
+* Add `-Xsource-features:double-definitions` to warn or error for double definitions ([#10846](https://github.com/scala/scala/pull/10846) by [@som-snytt](https://github.com/som-snytt))
+* Under `-Xsource:3`, allow importing `given`, for cross-building ([#10786](https://github.com/scala/scala/pull/10786) by [@som-snytt](https://github.com/som-snytt))
+* Under `-Xsource:3`, remove by-name restriction for case copy ([#10770](https://github.com/scala/scala/pull/10770) by [@som-snytt](https://github.com/som-snytt))
+
+### Errors and warnings
+
+* Fix `-Wconf` and `-nowarn` to be more consistent ([#10750](https://github.com/scala/scala/pull/10750) by [@som-snytt](https://github.com/som-snytt))
+* Add `-Wtostring-interpolated` to warn if interpolator uses `toString` ([#10776](https://github.com/scala/scala/pull/10776) by [@som-snytt](https://github.com/som-snytt))
+* Improve inferred-`Any` lint (fixing false positives and false negatives) ([#9452](https://github.com/scala/scala/pull/9452) by [@som-snytt](https://github.com/som-snytt))
+* Fix false-positive unused warnings in for comprehensions [ci: last-only] ([#10812](https://github.com/scala/scala/pull/10812) by [@som-snytt](https://github.com/som-snytt))
+* Fix false-positive by-name implicit warnings with `-Wmacros` ([#10781](https://github.com/scala/scala/pull/10781) by [@som-snytt](https://github.com/som-snytt))
+* Mention if missing symbol is on the class path ([#10774](https://github.com/scala/scala/pull/10774) by [@som-snytt](https://github.com/som-snytt))
+* Avoid unused import warnings in REPL ([#10769](https://github.com/scala/scala/pull/10769) by [@som-snytt](https://github.com/som-snytt))
+* Add `-Vcyclic` to improve reporting of "cyclic reference" errors ([#10680](https://github.com/scala/scala/pull/10680) by [@som-snytt](https://github.com/som-snytt))
+
+### Other improvements
+
+* REPL: JLine 3.26.3 (was 3.25.1) ([#10827](https://github.com/scala/scala/pull/10827)
+* Compiler plugins: rework and improve phase assembly (possibly requiring plugin authors to adjust) ([#10687](https://github.com/scala/scala/pull/10687) by [@som-snytt](https://github.com/som-snytt))
```

Comment:
maybe this is actually worth pulling up to "Breaking changes". Something like this:

> In projects using compiler plugins, this change may cause the plugin phases to be executed in a different order. If your project breaks or you just want to be sure, use the `-Vphases` flag before and after the update to 2.13.15 and compare the list of compiler phases.

Reply:
I think "no longer ignores broken constraints" is more likely to demand attention. A correct configuration does not change plugin order. 

The accommodation in scaladoc was for plugins "better monadic for has a constraint to run before patmat" with a constraint on patmat and erasure, which don't exist in scaladoc runs.

Also, mention my consulting rates.

Reply:
I see that in sbt,
```
set ThisBuild/scalacOptions += "-Vphases"
```
and compile.

Reply:
I agree to moving it up to Breaking changes.

I want to keep the text in the release announcement itself short. Let's keep it to:

>   * Plugin authors should check to see if they are affected
>  * Plugin users may see strange errors if a plugin is affected and not adjusted by its author

any longer explanations should go in the PR description, where it's fine to expound at any length. @som-snytt @lrytz Please edit the PR description to include any additional advice you want to give

Reply:
added a few words to the PR

Reply:
Thanks, Lukas. You bring structure and clarity. It's OK if you'd like to add, "If your project breaks, blame som-snytt." That is my joking way of appreciating how you both are mitigating risk through communication.